src/gengowatcher/cli.py

```
@@ -0,0 +1,157 @@
+        is_placeholder = current in PLACEHOLDER_CONFIG_VALUES
+
+        if is_placeholder:
+            label = "required" if required else "optional"
+            prompt_text = f"[label]{description}[/] [warning]({label})[/]: "
+        else:
+            masked = str(current)[:4] + "..." if len(str(current)) > 8 else str(current)
+            prompt_text = f"[label]{description}[/] [{masked}]: "
+
+        console.print(prompt_text, end="")
+        if option in {"user_key", "user_session"}:
+            new_value = getpass("").strip()
```

Ruff (0.15.5)

[error] Undefined name `getpass`

(F821)